pgcontents/alembic/env.py

```diff
@@ -15,10 +15,10 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-from os.path import dirname, abspath
-import sys
```

Comment:
The flake8 violation is about imports not at the top of the module. I assumed they were required to be after the call to ```sys.path.append```, but looks like it works either way, so rearranging...